gemrb/core/Store.cpp

```diff
@@ -135,6 +135,15 @@ int Store::AcceptableItemType(ieDword type, ieDword invflags, bool pc) const
 			ret &= ~IE_STORE_SELL;
 		}
 
+		//...the shop is special (iwd/how/bg1)
+		if ((core->HasFeature(GFFlags::NO_UNDROPPABLE)) || (core->HasFeature(GFFlags::BREAKABLE_WEAPONS))) {
+			if (invflags&IE_INV_ITEM_CONVERSABLE) {
+				ret &= ~IE_STORE_SELL;
+			} else {
+				ret |= IE_STORE_SELL;
```

Comment:
this could potentially enabling the sale to stores that don't allow it.

Reply:
also how should IE_INV_ITEM_DESTRUCTIBLE be handled in this case? Now you reenable their sellability.

Reply:
I added more exact gametype detection.

I'm guessing that the major concern here is getting new/modded games, outside the intended ones, caught in the selling factors?

Reply:
The gametype checks are there to ensure this code is only used for the games that need it. My comments in this thread are about the code itself — look at the rest of the function, potential execution flow.